src/main/kotlin/com/grepp/nbe1_3_team04/vote/domain/Choice.kt

```diff
@@ -0,0 +1,27 @@
+package com.grepp.nbe1_3_team04.vote.domain
+
+import jakarta.persistence.*
+
+@Table(
+    indexes = [
+        Index(name = "idx_choice_member_id", columnList = "memberId"),
+        Index(name = "idx_choice_vote_item_id", columnList = "voteItemId")
+    ]
+)
+@Entity
+class Choice private constructor(
+    val memberId: Long,
+    val voteItemId: Long
+) {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private val choiceId: Long? = null
```

Comment:
```suggestion
    memberId: Long,
    voteItemId: Long
) {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private val choiceId: Long? = null
    val memberId: Long = memberId
    val voteItemId: Long = voteItemId
```
저희 컨벤션 대로면 이렇게 작성해주시면 됩니다! 
다만 이렇게 쓰면 노란줄이 나옵니다..

Reply:
감사합니다 ~ 수정하겠습니다! 그렇네요 권장하는게 생성자쪽에서 진행하는건가봐여 일단 컨벤션대로 진행하겠습니다 !